### PR TITLE
Ux fixes

### PR DESCRIPTION
**Pull request by vanshBhatia-A4k9**
## Proposed changes
 - UI fixes
 - Pagination bug fix
 - Links changed to TextButtons
 - Removed un-used Redux state
 
Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/Analytics/Overview/index.tsx [hunk 1]
```diff
@@ -227,12 +221,13 @@ const Overview: React.FC = () => {
...
             <Link
               underline="none"
-              color="primary"
               href="https://prometheus.io/docs/introduction/overview/"
               target="_blank"
               rel="noreferrer"
             >
-              <Typography>Read prometheus doc</Typography>
+              <TextButton variant="highlight">
+                <Typography>Read prometheus doc</Typography>
```
Comment (amityt) [inline litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:229]: Can this be added in translation file?

**Review by amityt — CHANGES_REQUESTED** [review]

**Comment by amityt** [comment]
LGTM, just one minor change @vanshBhatia-A4k9.

File: litmus-portal/frontend/src/views/WorkflowDetails/workflowConstants.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ export const FAILED: string = 'Failed';
 export const RUNNING: string = 'Running';
 export const PENDING: string = 'Pending';
 export const SUCCEEDED: string = 'Succeeded';
+export const NOTAVAILABLE: string = 'NotAvailabe';
```
Comment (Saranya-jena) [inline litmus-portal/frontend/src/views/WorkflowDetails/workflowConstants.ts:5]: Can you fix this typo?

File: litmus-portal/frontend/src/views/Analytics/Overview/index.tsx (hunk 1 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:229]: All the translations for analytics will be raised as a separate PR as per earlier discussion @amityt

File: litmus-portal/frontend/src/views/WorkflowDetails/workflowConstants.ts (hunk 2 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/views/WorkflowDetails/workflowConstants.ts:5]: Will do, Thanks :+1:

File: litmus-portal/frontend/src/views/Analytics/Overview/index.tsx [hunk 3]
```diff
@@ -227,12 +221,13 @@ const Overview: React.FC = () => {
           link={
             <Link
               underline="none"
-              color="primary"
               href="https://prometheus.io/docs/introduction/overview/"
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:224]: Please update the Prometheus link, we do not have to point to official docs @vanshBhatia-A4k9

File: litmus-portal/frontend/src/views/Analytics/Overview/index.tsx [hunk 4]
```diff
@@ -227,12 +221,13 @@ const Overview: React.FC = () => {
...
               target="_blank"
               rel="noreferrer"
             >
-              <Typography>Read prometheus doc</Typography>
+              <TextButton variant="highlight">
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:228]: Docs

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/Analytics/Overview/index.tsx (hunk 3 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:224]: can you provide the appropriate link?

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:224]: https://github.com/litmuschaos/litmus/tree/master/monitoring#model-1-optional-prometheus-scrape-config-model

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/Analytics/Overview/index.tsx:224]: Use the above link @vanshBhatia-A4k9

**Review by SarthakJain26 — APPROVED** [review]

File: litmus-portal/frontend/src/views/Home/AgentConfiguredHome/index.tsx [hunk 5]
```diff
@@ -138,21 +138,19 @@ const AgentConfiguredHome: React.FC<AgentConfiguredHomeProps> = ({
...
-            <Link
-              underline="none"
-              color="primary"
+            <TextButton
+              variant="highlight"
               onClick={() => {
-                tabs.changeWorkflowsTabs(2);
                 history.push({
-                  pathname: '/workflows',
+                  pathname: '/myhub/Chaos%20Hub',
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/views/Home/AgentConfiguredHome/index.tsx:145]: is that space....ideally ChaosHub should a be single word right?

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/Home/AgentConfiguredHome/index.tsx (hunk 5 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/Home/AgentConfiguredHome/index.tsx:145]: @arkajyotiMukherjee  The `ChaosHub` in general is a term but here the `Chaos Hub ` is the name of the particular hub hence the space is required.

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

File: litmus-portal/frontend/src/views/WorkflowDetails/workflowConstants.ts (hunk 2 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/WorkflowDetails/workflowConstants.ts:5]: This is not resolved yet.

**Review by Saranya-jena — APPROVED** [review]

**Pull request merged**